.config/.terraform-docs.yml

```diff
@@ -27,12 +27,26 @@ content: |-
   3. Navigate to the AWS Chatbot Service in the AWS Console and authorize the Slack Workspace.
   4. It may fail to deploy the first time due to an underlying AWS config replication, if it does wait up to 15 minutes as the error message states.
 
+  ## Notes
+  - This module assumes that the Terraform state file is located in the account that you will be acting in. It has NOT been tested with cross account state file
+  - This module is designed to work from a tooling account and authenticate to destination accounts via cross account IAM roles. In theory it could run from a single account with another role. This has yet to be tested.
+  - This module can pull in code from either a code star connection or via an s3 artifact dump.
```

Comment:
Suggestion: 

- This module can pull in code via either an AWS CodeStar connection or an artifact in an S3 bucket.

Reply:
ef0c5d22ce9fffa509595afcaf2713e4b54bb79b